### PR TITLE
Lab9

### PR DESCRIPTION
**Pull request by Maxkoz777**

### CODE REVIEW

**Review by afonyaa — COMMENTED** [review]
Not sure, but probably, directory names indicate Nodes, not Pods.

File: k8s/python_pod/deployment.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: python-application-deployment
+  labels:
+    app: nginx
```
Comment (afonyaa) [inline k8s/python_pod/deployment.yml:6]: why nginx?

**Comment by afonyaa** [comment]
Amazing!!!!!

**Comment by none-word** [comment]
Nice!

**Comment by AKurmazov** [comment]
Well done 👍

**Comment by Homa3030** [comment]
👍🏻

**Pull request closed without merging**